src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -130,14 +131,49 @@ private CallbackResponse validateApplicantExperts(CallbackParams callbackParams)
     }
 
     private CallbackResponse setApplicant1ProceedFlag(CallbackParams callbackParams) {
-        CaseData caseData = callbackParams.getCaseData();
+        CaseData updatedCaseData = setApplicant1ProceedFlagToYes(callbackParams.getCaseData());
```

Comment:
Can you try to create a 2v1 case (LR,LR v LR) with same response to see what happens after your change of the code

Reply:
tested.. working as expected.